pkgs/games/keeperrl/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub, SDL2, SDL2_image, openal, libvorbis, zlib, curl,
+  libGL, makeWrapper, boost }:
+
+stdenv.mkDerivation rec {
+  name = "keeperrl-${version}";
+  version = "alpha24";
+
+  src = fetchFromGitHub {
+    owner = "miki151";
+    repo = "keeperrl";
+    rev = "${version}";
+    sha256 = "00vpq4njdl8hqndjvx2r0w0azz9lfil4qg1rlhj3y9qvc1i0xwwn";
+  };
+
+  patches = [ ./makefile_cflags.patch ];
+
+  buildInputs = [
+    SDL2 SDL2_image openal libvorbis zlib curl libGL makeWrapper
+  ] ++ stdenv.lib.optional stdenv.isDarwin boost ;
+
+  makeFlags = [ "OPT=true" "RELEASE=true" ];
+
+  CFLAGS = "-I${SDL2.dev}/include/SDL2";
+
+  installPhase = ''
+    mkdir -p $out/keeperrl
+    mkdir -p $out/bin
+    cp ./keeper $out/bin/
+    cp -r data_contrib data_free  $out/keeperrl
+    wrapProgram $out/bin/keeper \
+      --run 'mkdir -p $HOME/.keeperrl' \
+      --run 'cd $HOME/.keeperrl' \
+      --run "rm data_free || true" \
+      --run "rm data_contrib || true" \
+      --run "ln -s $out/keeperrl/data_free ." \
+      --run "ln -s $out/keeperrl/data_contrib ."
```

Comment:
I don't think these hacks are needed anymore. See https://github.com/miki151/keeperrl/issues/376#issuecomment-71612420 and https://github.com/miki151/keeperrl/pull/568

Reply:
Changed it for both DATA_DIR and USER_DIR. I just wonder where should be a note somewhere for users explaining what to do if they have the paid data dir.